contrib/packs/actions/download.yaml

```diff
@@ -5,23 +5,15 @@
   enabled: true
   entry_point: "pack_mgmt/download.py"
   parameters:
-    packs:
-      type: "array"
-      items:
-        type: "string"
-    repo_url:
+    name_or_url:
       type: "string"
-      default: "StackStorm/st2contrib"
-    branch:
+    ref:
```

Comment:
Yeah, `ref` is pretty much a working title. I like `version` the most, honestly, but to use this word we need to make people follow at least some basic convention with branching/tagging. :) Think `version` would be ok? I'm fine with `gitref`, too, but it's slightly more confusing for new users.